gnucash/gnome/reconcile-view.c

```diff
@@ -901,13 +901,27 @@ void
 gnc_reconcile_view_refresh (GNCReconcileView *view)
 {
     GNCQueryView *qview;
+    GtkTreeSelection *selection;
+    GList *path_list, *node;
 
     g_return_if_fail (view != NULL);
     g_return_if_fail (GNC_IS_RECONCILE_VIEW (view));
 
     qview = GNC_QUERY_VIEW (view);
     gnc_query_view_refresh (qview);
 
+    /* Ensure last selected split, if any, can be seen */
+    selection = gtk_tree_view_get_selection (GTK_TREE_VIEW (qview));
+    path_list = gtk_tree_selection_get_selected_rows (selection, NULL);
+    node = g_list_last(path_list);
```

Comment:
can you add a space after g_list_last

Reply:
Fixed in PR 707 (against master). I've implemented your method in gnc_reconcile_window_delete_set_next_selection() which pre-selects the next split before the deletion.